test/qa/test_meshnet.py

```diff
@@ -182,6 +184,7 @@ def test_set_meshnet_off_when_logged_out(meshnet_allias):
     assert "You are not logged in." in ex.value.stdout.decode("utf-8")
 
 
+@pytest.mark.xfail(condition=DISABLE_MESHNET_TESTS, reason="Run only in nightly")
```

Comment:
test previously was not marked as xfail, now is, so it's a flaky test that will be run and not xfail in our day to day pipeline due to core meshnet mark

is this the intention?